### ISSUE TITLE
Nintendo Switch crashes up server list loading

### ISSUE THREAD

**Issue by Lovingtaken3** [Pugmatt/BedrockConnect#510]
### What happened?

As soon as I load and and the server list opens, the game crashes

### Expected Behaviour?

Game shouldn't crash

### Steps to reproduce.

1.Join with DNS or MCXboxBroadcast instance on Switch.
2. Loading into BedrockConnect instance
3. Crashes

### Minecraft Bedrock Version

1.21.1

### Console

Nintendo Switch

### Additional Context

Seems to only happen on Switch, dont have another working switch to see if its only my switch.

**Comment by Lovingtaken3** [comment]
This has been confirmed to work on any switch, my players are complaining they crashed after joining the server list on switch

**Comment by Pugmatt** [comment]
I attempted to replicate this on my Switch, but it loaded the serverlist fine as usual without crashing. 

Which BedrockConnect instance are you using?

**Comment by Lovingtaken3** [comment]
I am using my own instance I hosted as described. I made sure that it's on the latest update for Bedrock Connect.

**Comment by Lovingtaken3** [comment]
I can record a video when I get home

**Comment by Pugmatt** [comment]
What settings do you have enabled on your BedrockConnect instance? (Language file, custom servers, etc) Just trying to see if I can replicate it.

Does this crash happen everytime, or does it happen only sometimes?

**Comment by Lovingtaken3** [comment]
https://github.com/Pugmatt/BedrockConnect/assets/17980552/22f13062-db65-48b1-a4ea-bdb5aa045be9

**Comment by Lovingtaken3** [comment]
> Thanks, will await the video. Couple of further questions, so I can troubleshoot this easier:
> 
> What settings do you have enabled on your BedrockConnect instance? (Language file, custom servers, etc) Just trying to see if I can replicate it.
> 
> Does this crash happen everytime, or does it sometimes let you join without crashing?

Yes, I have custom settings, here is the command I use, java -Xms1G -Xmx1G -jar BedrockConnect-1.0-SNAPSHOT.jar custom_servers=servers.json mysql_host=play.mcrashcraft.com mysql_db=bedrock_connect mysql_user=MCrashCraft mysql_pass=Conaway12162001 language=mcrashcraft.json, my language config: 
[mcrashcraft.json](https://github.com/user-attachments/files/16150638/mcrashcraft.json) and my custom server config: 
[servers.json](https://github.com/user-attachments/files/16150647/servers.json)

**Comment by Lovingtaken3** [comment]
I am also using MCXboxBroadcast

**Comment by Lovingtaken3** [comment]
So I got rid of some custom servers and now it works perfectly
